### ISSUE TITLE
Surprising message from bookmark function

### ISSUE THREAD

**Issue by matself** [hajkmap/Hajk#736]
**Describe the bug**
When using the Bookmark function, the user is advised:
"Alla lager i kartan kommer nu att släckas."
which made me click again, thinking I had hit the wrong icon.

**To Reproduce**
1. Use HAJK 3.5 release, standard map
2. Click on 'Snabbval'
3. Observe message.

**Expected behavior**
Perhaps "Kartan flyttas till [bookmarkname]"
And make the user decide whether to close map layers or not.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Any
 - Version 3.5.0

**Smartphone (please complete the following information):**
n/a

**Additional context**
IMO, the caption should be 'Bokmärke' instead of 'Snabbval'. I think it is a more common Swedish term for such a function.

**Comment by jacobwod** [comment]
There's no guarantee that all layers available in map A will also be available in map B, hence the warning. It's not an option to "let the user decide" as you propose, because map configs are not a user setting.

**Comment by matself** [comment]
I disagree. In normal use, it would be perfectly valid and expected that visible layers are retained when moving between bookmarks in close vicinity, or within the scope of the organisation.
A municipality would have bookmarks for its towns and villages.
A forestry company would have bookmarks for its holdings or current logging.

**Comment by jesade-vbg** [comment]
Just to clarify. I believe the tool in this discussion is the "Snabbval" tool (Named preset in code). The Bookmark tool is intended for the user to create bookmarks. The Snabbval is preset bookmarks intended for the municipality/business to configure.

I have never heard of a bookmark functionality that gives the user the option to exclude/include parameters. In this case, should we also let the user decide....... Would you like to Zoom? Would you like to Pan? Probably not, because the name "Snabbval" would no longer apply (not quick anymore).

If I would suggest anything, I would suggest to remove the warning.
